source/extensions/common/wasm/context.cc

```diff
@@ -504,6 +504,9 @@ Context::FindValue(absl::string_view name, Protobuf::Arena* arena) const {
       return CelValue::CreateString(&info->upstreamHost()->cluster().name());
     } else if (info && info->routeEntry()) {
       return CelValue::CreateString(&info->routeEntry()->clusterName());
+    } else if (info && info->upstreamClusterInfo().has_value() &&
```

Comment:
Maybe make this the first clause since it's going to become the main way?

Reply:
For 1.6? don't want to add a new thing as main way to 1.5?